bigchaindb/common/utils.py

```diff
@@ -92,9 +92,28 @@ def validate_all_keys(obj_name, obj, validation_fun):
     """
     for key, value in obj.items():
         validation_fun(obj_name, key)
-        if type(value) is dict:
+        if isinstance(value, dict):
             validate_all_keys(obj_name, value, validation_fun)
-    return
+
+
+def validate_all_value_for_key(obj, key, validation_fun):
+    """Validates value for all (nested) occurences of `key` in `obj`
+       using `validation_fun`
+
+        Args:
+            obj (dict): dictonary object.
```

Comment:
dictonary → dictionary

Reply:
sad to see myself making these mistakes !